tests/templates/test_subroutines/test_mod_exp.py

```diff
@@ -136,6 +145,18 @@ def test_wires_error(
         with pytest.raises(ValueError, match=msg_match):
             qml.ModExp(x_wires, output_wires, base, mod, work_wires)
 
+    def test_check_base_and_mod_are_coprime(self):
+        """Test that an error is raised when base and mod are not coprime"""
+
+        with pytest.raises(ValueError, match="base has no inverse modulo mod"):
```

Comment:
I think this solve the current issue 

```suggestion
        with pytest.raises(ValueError, match="The operator cannot be built because base has no inverse modulo mod."):
```

Reply:
The issue was because there was a test in the `capture` tests that created an invalid operator (base=2, mod=None), it works now :)